test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
As in:

Given the following `basic.proto`:

```proto
package MyPackage;

message Basic {
  required string basic = 1;
}
```

And the following usage:

```elixir
defmodule MyTopLevelModule
  use Protobuf, "basic.proto"
end

defmodule NotMyTopLevelModule
  use Protobuf, "basic.proto", inject: true
end
```

=== Before This PR We Get

```elixir
MyTopLevelModule.Basic.new
Basic.new
```

=== After This PR We Get
```elixir
MyPackage.MyTopLevelModule.Basic.new
MyPackage.Basic.new
```